### PR TITLE
feat(dynamic-sampling): Add isSupportedPlatform to dynamic-sampling SDK versions endpoint [TET-307]

### PR DESCRIPTION
**Pull request by andriisoldatenko**
Some SDKs do not support sampling yet we show prompts that
they should update to the latest version. We check sdk_sname
against allowlist of supported SDKs and  only prompt to update
those which are in the list.

### CODE REVIEW

File: src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py [hunk 1]
```diff
@@ -15,6 +15,72 @@
 SDK_NAME_FILTER_THRESHOLD = 0.1
 SDK_VERSION_FILTER_THRESHOLD = 0.05
 
+# Some SDKs do not support sampling yet,
+# we show prompts that they should update to the latest version.
+# Allowlist of supported SDKs and only prompt to update those.
+ALLOWED_SDK_NAMES = frozenset(
+    (
+        "sentry.javascript.browser",  # JavaScript Browser
```
Comment (andriisoldatenko) [inline src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py:23]: we need clarify this list.

Comment (andriisoldatenko) [reply src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py:23]: @jan-auer can you please help to full fill this list?

File: src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py [hunk 2]
```diff
@@ -15,6 +15,27 @@
...
+        "sentry.javascript.angular",  # Angular
+        "sentry.javascript.ember",  # Ember
+        "sentry.javascript.vue",  # Vue.js
+        "sentry.javascript.nextjs",  # Next.js
+        "sentry.javascript.remix",  # RemixJS
+        "sentry.javascript.node",  # Node, Express, koa
+        "sentry.javascript.serverless",  # AWS Lambda Node TODO: need to double check
+        "sentry.python",  # python, django, flask, FastAPI, Starlette, Bottle, Celery, pyramid, rq
+        "sentry.python.serverless",  # AWS Lambda TODO: need to double check
+        "sentry.cocoa",  # iOS
```
Comment (andriisoldatenko) [inline src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py:34]: @bruno-garcia could you please review this list 🙏 ?

**Review by ahmedetefy — COMMENTED** [review]
code LGTM! not sure about the list so I will let others go over that

**Comment by andriisoldatenko** [comment]
> code LGTM! not sure about the list so I will others go over that

yeah let's wait for final review of list from @jan-auer  / @bruno-garcia

**Review by matejminar — APPROVED** [review]
Looks good to me. We can add more SDKs if needed in a follow-up PR.

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Pull request merged**